web/src/app/monitors/monitor-create-edit/monitor-create-edit.component.html

```diff
@@ -0,0 +1,54 @@
+<div *ngIf="monitorForm" class="mat-content" fxLayout="column" fxLayoutGap="20px">
+  <form [formGroup]="monitorForm">
+    <mat-card>
+      <mat-card-header>
+        <mat-card-title> <span *ngIf="!isEdit">Create</span> <span *ngIf="isEdit"> Edit </span> Monitor</mat-card-title>
```

Comment:
Can this be split on multiple lines to be read more easily

Reply:
done